requirements.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-backports.ssl-match-hostname==3.5.0.1  # via docker-py
+backports.ssl-match-hostname==3.5.0.1  # via docker-py, websocket-client
```

Comment:
Very minor thing, but don't think it's related with other changes in the PR:
it only means that [your python version is older than `2.7.10`](https://github.com/websocket-client/websocket-client/blob/master/setup.py#L15-L21).

Reply:
You're right, I'll fix this.